jsk_apc2016_common/config/bin_shelf.yaml

```diff
@@ -0,0 +1,14 @@
+initial_pos_list_bottom: [[-0.27, 0.25, -0.41], [-0.24, -0.01, -0.41], [-0.24, -0.32, -0.41], [-0.27, 0.25, -0.12], [-0.24, -0.01, -0.12], [-0.24, -0.32, -0.12]]
+initial_rot_list_bottom: [[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]]
```

Comment:
it is not good for 2015 to depend on 2016.
you should use arg of roslaunch to pass the yaml file.